tests/appsec/iast/sink/test_sql_injection.py

```diff
@@ -26,7 +26,6 @@
         "resteasy-netty3": "1.11.0",
         "jersey-grizzly2": "1.11.0",
         "vertx3": "1.12.0",
-        "akka-http": "1.12.0",
```

Comment:
Why? I think this is an error (disabling this for SQL Injection).

Reply:
yes, error

Reply:
fixed